pkg/providers/common/common.go

```diff
@@ -0,0 +1,8 @@
+package common
+
+import (
+	_ "embed"
+)
+
+//go:embed config/audit-policy.yaml
+var AuditPolicy string
```

Comment:
Can we make this a private variable and have a corresponding function that returns the contents? 

```suggestion
var auditPolicy string

func AuditPolicy() string {
    return auditPolicy
}
```

Reply:
Done, good callout.